docs/digger/installation/ansible-installs-jenkins.asciidoc

```diff
@@ -1,9 +1,10 @@
-=== How Ansible Installs Jenkins
+=== Upload User Public Keys for Automatic Jenkins Installation
 //pwright: I think heading should be task oriented, ie run playbook
 
-The installer will perform the following tasks for Jenkins setup and configuration:
+The installer creates the Jenkins service on OpenShift with required routes, services and pods, and the waits for Jenkins to be running. It also creates an admin user.
+//Admin user where ? ^
```

Comment:
pls resolve

Reply:
I dont think this is accurate. The installer creates the Jenkins instance and route but I don't believe we create an admin user ping @odra for clarification

Reply:
Correct, the installer assumes that the admin/jenkins user was already created with, we have a sample bash script that creates this user: [https://github.com/aerogear/digger-jenkins/blob/master/admin/create-osx-user.sh](https://github.com/aerogear/digger-jenkins/blob/master/admin/create-osx-user.sh)

Note: The mac machine may need to be rebooted after creating this user.